src/main/java/build/buildfarm/common/DigestUtil.java

```diff
@@ -91,6 +91,13 @@ private ActionKey(Digest digest) {
   private final HashFunction hashFn;
   private final Digest empty;
 
+  public static DigestUtil forHash(String hashName) {
```

Comment:
Just curious, where do you anticipate needing this function?

Reply:
`bf-cat` argument interpretation

Reply:
Also the other `bf-*` utils that communicate with instances.